test/TestUtilities/Test.Utility/TestServer/TcpListenerServer.cs

```diff
@@ -53,6 +70,50 @@ public async Task<T> ExecuteAsync<T>(Func<string, Task<T>> action)
                 CancellationToken.None);
         }
 
+        public TcpListenerServer()
+        { }
+
+        private static X509Certificate2 GenerateSelfSignedCertificate()
```

Comment:
`GenerateSelfSignedCertificate` method has same logic in this class and also `SelfSignedCertificateMockServer`. If possible, please consider removing the code duplication in a follow-up PR.